types/convict-format-with-validator/convict-format-with-validator-tests.ts

```diff
@@ -0,0 +1,22 @@
+import convictValidators = require('convict-format-with-validator');
+
+// $ExpectType string | undefined
+convictValidators.email.name;
+// $ExpectType ((val: any) => void) | undefined
```

Comment:
This type isn't any, is the return value from the convict: 
https://github.com/mozilla/node-convict/blob/95f4ab398719d002a81b8acb21d2317b7bdbc8b6/packages/convict/src/main.js#L458-L703